files/es/learn/common_questions/thinking_before_coding/index.html

```diff
@@ -74,35 +74,35 @@ <h3 id="¿Qué_es_exactamente_lo_que_quiero_lograr">¿Qué es exactamente lo que
 
 <ol>
  <li>Encontrar un/a nuevo/a novio/a.</li>
- <li>Permitir que las gente escuche tu música.</li>
+ <li>Posibilitar que la gente escuche tu música.</li>
  <li>Hablar acerca de tu música.</li>
- <li>Conocer otros músicos.</li>
- <li>Venta de golosinas.</li>
+ <li>Conocer a otros músicos.</li>
+ <li>Vender golosinas.</li>
  <li>Enseñar música a través de vídeos.</li>
  <li>Publicar fotos de tus gatos.</li>
 </ol>
 
-<p>Haciendo este sencillo ejercicio, escribir metas y clasificarlas, te ayudará cuando tengas que tomar decisiones. (¿Debería implementar estas características, usar estos servicios, crear este diseño?)</p>
+<p>Hacer este sencillo ejercicio, escribir metas y clasificarlas, te ayudará cuando tengas que tomar decisiones. (¿Debería implementar estas características, usar estos servicios, crear este diseño?)</p>
 
-<p>Entonces, ahora que tienes tu lista de objetivos priorizada, pasemos a la siguiente pregunta.</p>
+<p>Entonces, ahora que tienes tu lista de objetivos priorizados, pasemos a la siguiente pregunta.</p>
 
 <h3 id="¿Cómo_podría_un_sitio_web_llevarme_a_mis_objetivos">¿Cómo podría un sitio web llevarme a mis objetivos?</h3>
 
 <p>Entonces, tienes tu lista de objetivos y sientes que necesitas un sitio web para alcanzarlos. ¿Estás seguro?</p>
 
 <p>Echemos un vistazo a nuestro ejemplo. Tenemos cinco metas relacionadas con la música, una relacionada a la vida personal (Encontrar pareja), y las fotos del gato completamente ajenas. ¿Es razonable construir un único sitio para cubrir todos esos objetivos? ¿Es necesario? Después de todo, una gran cantidad de servicios web existentes podría llevarte a tus objetivos sin necesidad de crear uno nuevo.</p>
 
-<p>Encontrar pareja es un caso prioritario en el que tiene más sentido utilizar los recursos existentes en lugar de construir todo un nuevo sitio. ¿Por qué? Porque invertiríamos más tiempo construyendo y manteniendo el sitio web en lugar de realmente buscar una pareja. Dado que nuestro objetivo es lo que más importa, debemos gastar nuestra energía en aprovechar las herramientas ya existentes en lugar de empezar desde cero. Una vez más, hay tantos servicios web ya disponibles para mostrar fotos que no vale la pena el esfuerzo para construir un nuevo sitio sólo para anunciar acerca de lo lindo que son nuestros gatos.</p>
+<p>Encontrar pareja es un caso prioritario en el que tiene más sentido utilizar los recursos existentes, en vez de construir todo un nuevo sitio. ¿Por qué? Porque invertiríamos más tiempo construyendo y manteniendo el sitio web en lugar de realmente buscar una pareja. Dado que nuestro objetivo es lo que más importa, debemos gastar nuestra energía en aprovechar las herramientas ya existentes en lugar de empezar desde cero. Nuevamente, hay tantos servicios web ya disponibles para mostrar fotos que no vale la pena el esfuerzo para construir un nuevo sitio sólo para anunciar acerca de lo tiernos que son nuestros gatos.</p>
```

Comment:
* no se habla de algo nuevo

```suggestion
<p>Encontrar pareja es un caso prioritario en el que tiene más sentido utilizar los recursos existentes, en vez de construir todo un nuevo sitio. ¿Por qué? Porque invertiríamos más tiempo construyendo y manteniendo el sitio web en lugar de realmente buscar una pareja. Dado que nuestro objetivo es lo que más importa, debemos gastar nuestra energía en aprovechar las herramientas ya existentes en lugar de empezar desde cero. Una vez más, hay tantos servicios web ya disponibles para mostrar fotos que no vale la pena el esfuerzo para construir un nuevo sitio sólo para anunciar acerca de lo tiernos que son nuestros gatos.</p>
```